pyproject.toml

```diff
@@ -6,8 +6,8 @@ authors = ["Cristian Sirbu <cristian.sirbu@networktocode.com>", "Leo Kirchner <l
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-pylint = "^2.13"
+python = "^3.7.2"
+pylint = "^2.17"
```

Comment:
I was temped to do the same from the get go, but it will clash when adding into other projects that use `^3.7`, which is all plugins baked from our cookie. I think we need to stick to the old one until we bump minimum python to the next minor.

Reply:
what would you like me to do with this PR then?

Reply:
Hold for a bit. Maybe we can raise the floor at least for 2.0 onwards. There's a sizeable delta in pylint release that all of our projects can't use because of this 3.7.

Reply:
Might be able to do `poetry add pylint^2.17 --python ^3.7.2` here, we did [something similar](https://github.com/nautobot/nautobot/blob/develop/pyproject.toml#L201) in core